comeet/UI/Login/LoginViewModel.swift

```diff
@@ -40,7 +40,7 @@ class LoginViewModel : BaseViewModel {
     }
     
     func isLoggedIn() -> Bool {
-        return authenticator.isLoggedIn()
+        return true //authenticator.isLoggedIn()
```

Comment:
I think we need to get back to authenticator.isLoggedIn(). I'm assuming this was for testing purposes.